### PR TITLE
Home page slider responsive fixes

### PR DESCRIPTION
**Pull request by oleksify**
Hello, guys.

Recently I discovered that pivorak.com main page slider is not responsive. Also, on my MacBook 13" part of it was entirely hidden so it's hard to read the content of third speaker (or whatever is there in the green card): 

So, I decided to add some fixes. It's open source anyway, right? I discovered that most of the stuff had hard coded sizes in units. So, I had to rewrite a chunk of code for the slides and change the way geometry shapes are built in CSS. Also, typography is kind of fluid too. But I think there is more to be done to make this even easier to read and more accessible.

As the result, the color cards are completely fluid and responsive:

These days I mostly use `rem` units in my code, If you decided to stick to only pixels, I can rewrite it. 

Let me know what you think!

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -549,7 +549,7 @@ DEPENDENCIES
   yt (~> 0.30.1)
 
 RUBY VERSION
-   ruby 2.3.1p112
+   ruby 2.4.0p0
```
Comment (denys-medynskyi) [inline Gemfile.lock]: Can you do it in separate PR? we need to update ruby version on `production` before doing that. Also it is not related to `ui-fixes`

Comment (oleksify) [reply Gemfile.lock]: @denysmedynskyi Sorry, my mistake. Fixed.

**Review by denys-medynskyi — APPROVED** [review]
Tested locally. 👍 
However, I am not frontend guy)
I will request review from frontenders and remind in the slack.

**Review by pbilyk — APPROVED** [review]

**Comment by Okbil** [comment]
I like it :)

**Comment by denys-medynskyi** [comment]
Locally tests are passing. Merging it,
